core/src/main/kotlin/org/evomaster/core/search/gene/collection/TupleGene.kt

```diff
@@ -170,7 +170,7 @@ class TupleGene(
         val i = it.getValueAsRawString()
         "${it.name} : $i"
     } else {
-        if (it is ObjectGene || (it is OptionalGene && it.gene is ObjectGene)) {
+        if (it is ObjectGene || (it.getWrappedGene(OptionalGene::class.java)?.gene  is ObjectGene)) {
```

Comment:
this seems redundant. could directly check for `it.getWrappedGene(ObjectGene::class.java) != null`